### ISSUE TITLE
Creation of new detox tests for home screen personalization - Space Force support

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8103]
This ticket is not for confirming existing tests still work

This ticket is for creating new detox tests which should cover:
* Spaceforce icon support

_Other new detox test work split out into ticket #8915 (which I will further split as needed) on 6.26.24._

**Comment by TKDickson** [comment]
Deeplink buttons:
* Buttons exist
* Tapping buttons deep links to the correct screen
* (still TBD if this will be achieved in a way that's detox-testable - taking action updates the button counts; buttons disappear when count is 0)

VSC will actually be part of #8102, since it's an update to an existing component

DR/$
* Exist

Announcement banner
* Exists
* When tapped, brought to correct website (screenshot?)

Category screens
* Information exists (for ones where we've mocked)
* do NOT need to add explicit "go to correct place when tapped" because that's explicitly tested as part of getting to the features themselves for testing

**Comment by TKDickson** [comment]
Yeah - no. Everything is blocked by the cerner demo mode work; belatedly also moving this ticket to blocked.

**Comment by TKDickson** [comment]
@theodur I think I was (mentally) making your question more complicated than it needed to be. Thank goodness.

If you create a new "personalization" (or similarly named) script, and put working tests for any of these things into it, I think that'd be the easiest to split out:
* Activity section/Deeplink buttons
* About you section/Disability & payments
(Arguably the veteran status card tests currently on the home screen script, should actually go into this new script. If you end up taking this part, and making that change, let me know)

You could also add spaceforce icon handling to the existing MilitaryInformation screen.

I'd really rather that you mostly avoid the home screen & category screens, since I'm deeply into changes for those for #8102.

**Comment by theodur** [comment]
@TKDickson That all sounds good. I'll start with the Spaceforce icon test, since that one doesn't touch the home screen & health screen

**Comment by TKDickson** [comment]
OK split out the other work into a new ticket; this one is now just space force. Thanks @theodur

**Comment by theodur** [comment]
PR for this has been merged ✅
